docs/GREYFOX_ISA.txt

```diff
@@ -279,57 +300,57 @@ One Operand Class:   [ 3:0 A  ] [ 15:4 op2 [ values 0x010-0x0FF]]
 
         0580-05FF unused
     add_sub_pow2_constant: [A=i0-i15]
-        060x add A+=2
```

Comment:
I was hoping these all would look like just one "add/sub constant" mnemonics in the ISA. 

ie. 

add i0, 10
add i0, 25
add f0, 0.25

Is something like that possible instead of having separate mnemonics for each constant?

Reply:
The thing is that it's hard to encode "these particular values are allowed" as a user experience thing for a person writing assembly. Plus I have my hands full writing an assembler/compiler-backend in general. If I figure out how to do it later on, I can trivially make an alias to keep allowing the old wording and make your suggestion the default, but I *really* want to keep one instruction = 1 opcode for now. I know of other arches that use special values like this, but they're usually in register position instead of opcode position.

This isn't the only point where how the assembler's thoughts on the isa differs from the spec. For example, it may turn out to be easier to encode atomic-args+atomic-op as one 32b psuedo instruction. I just want to make sure that if there is an assembly instruction name, it's in the ISA spec.

Reply:
Okay, that seems fair. 

WRT, atomic-args+op, a single 32b psuedo instruction was how I was thinking this would work too. 

Everything else looks good to me. So, I'll merge this in.